### PR TITLE
docs: added consul infographic

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds a "Why Consul" section to the introduction page and incorporates the Consul infographic for operator/developer persona.

### CODE REVIEW

**Comment by pcmccarron** [comment]
LGTM

**Review by pcmccarron — APPROVED** [review]

File: website/content/docs/intro/index.mdx [hunk 1]
```diff
@@ -16,9 +16,19 @@ with Consul. We cover what Consul is, what problems it can solve, how it compare
...
 [HashiCorp Learn tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-install).
 
+## Why Consul?
+
+Consul solves the challenges that organizations of all sizes encounter with microservices architectures. This ranges from operating in various distributed environments and geographical locations, to meeting the need of securing all application traffic.
+The world is rapidly changing and evolving, so is the computing networking layer.
+
+Today's network must quickly adapt and ensure communication is encrypted at all times. Consul enables organizations to embrace a [zero trust](https://www.hashicorp.com/solutions/zero-trust-security) model while scaling up.
+Consul can achieve all this while reducing the burden on both operators and developers through automation of crucial networking tasks
+
```
Comment (trujillo-adam) [inline website/content/docs/intro/index.mdx:29]:
```suggestion
Consul solves the computing network layer challenges that organizations of all sizes face when implementing microservice architectures. These challenges range from operating in various distributed environments and geographical locations to securing all application traffic.

Modern networks must be highly adaptable while ensuring that communication is always encrypted. Consul enables organizations to embrace a [zero trust](https://www.hashicorp.com/solutions/zero-trust-security) model that scales to meet fluctuating workloads.
Consul achieves these goals while reducing the burden on operators and developers by automating crucial networking tasks.

```
Ignore this suggestion if it doesn't seem like an improvement --  experimenting with the language to see if we can improve the flow.

**Review by trujillo-adam — APPROVED** [review]
Made a few suggestions

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/536346.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e5a0c397d469a9d742f7a6ded342bc185b8c3684 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e5a0c397d469a9d742f7a6ded342bc185b8c3684 onto `release/1.11.x` succeeded!

**Pull request merged**